File: VB/NavigationDemo.Wpf/ViewModels/NavigationPane/ContactsNavigationViewModel.vb
Imports System.Collections.ObjectModel
Imports DevExpress.Mvvm.POCO

Namespace NavigationDemo
    Public Class NavigationViewModelBase
    End Class

    Public Class ContactsNavigationViewModel
        Inherits NavigationViewModelBase

        Public Shared Function Create() As ContactsNavigationViewModel
            Return ViewModelSource.Create(Function() New ContactsNavigationViewModel())
        End Function
        Public Overridable Property Contacts() As ObservableCollection(Of ContactItem)

        Protected Sub New()
            Contacts = NavigationPaneData.ContactsData
        End Sub
    End Class
End Namespace

File: CS/NavigationDemo.Wpf/ViewModels/NavigationPane/ContactsNavigationViewModel.cs
using System.Collections.ObjectModel;
using DevExpress.Mvvm.POCO;

namespace NavigationDemo {
    public class NavigationViewModelBase { }

    public class ContactsNavigationViewModel : NavigationViewModelBase {
        public static ContactsNavigationViewModel Create() {
            return ViewModelSource.Create(() => new ContactsNavigationViewModel());
        }
        public virtual ObservableCollection<ContactItem> Contacts { get; set; }

        protected ContactsNavigationViewModel() {
            Contacts = NavigationPaneData.ContactsData;
        }
    }
}
